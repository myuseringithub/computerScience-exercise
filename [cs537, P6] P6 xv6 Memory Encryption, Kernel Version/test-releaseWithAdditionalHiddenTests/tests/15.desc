[16] Basic test: Allocate multiple pages which will fill up the clock queue without any eviction with CLOCKSIZE = 16
