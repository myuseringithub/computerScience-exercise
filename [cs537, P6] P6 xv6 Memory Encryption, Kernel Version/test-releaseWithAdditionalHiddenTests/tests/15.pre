rm -rf src/param_old.h;
mv src/param.h src/param_old.h;
sed -E 's/((^|(\t| ))*CLOCKSIZE)(\t| )*[^ ]*/\3CLOCKSIZE\t16/' src/param_old.h > src/param.h;
make -f Makefile.test clean > /dev/null 2>&1;
make -f Makefile.test xv6.img > /dev/null 2>&1;
make -f Makefile.test fs.img > /dev/null 2>&1;
