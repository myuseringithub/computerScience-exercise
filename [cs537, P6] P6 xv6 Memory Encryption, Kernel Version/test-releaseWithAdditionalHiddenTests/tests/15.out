XV6_TEST_OUTPUT Index 0: pdx: 0x0, ptx: 0x10, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 1: pdx: 0x0, ptx: 0xF, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 2: pdx: 0x0, ptx: 0xE, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 3: pdx: 0x0, ptx: 0xD, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 4: pdx: 0x0, ptx: 0xC, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 5: pdx: 0x0, ptx: 0xB, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 6: pdx: 0x0, ptx: 0xA, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 7: pdx: 0x0, ptx: 0x9, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 8: pdx: 0x0, ptx: 0x8, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 9: pdx: 0x0, ptx: 0x7, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 10: pdx: 0x0, ptx: 0x6, writable bit: 1, encrypted: 0, ref: 1
